src/versions/develop/pages/BO/catalog/products/index.ts

```diff
@@ -769,7 +769,10 @@ class ProductsPage extends BOBasePage implements BOProductsPageInterface {
    */
   async resetFilter(page: Page): Promise<void> {
     if (!(await this.elementNotVisible(page, this.filterResetButton, 2000))) {
-      await this.clickAndWaitForURL(page, this.filterResetButton);
+      // Move the mouse to avoid the tooltip on first row
+      await page.mouse.move(0, 0);
+      await this.clickAndWaitForLoadState(page, this.filterResetButton);
+      await this.elementNotVisible(page, this.filterResetButton, 2000);
```

Comment:
`elementNotVisible`Return only true or false, it don't check that the element is not visible